config/manager/manager.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+    app.kubernetes.io/name: namespace
+    app.kubernetes.io/instance: system
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: opendatahub-operator
+    app.kubernetes.io/part-of: opendatahub-operator
```

Comment:
created-by is deprecated : https://kubernetes.io/docs/reference/labels-annotations-taints/#app-kubernetes-io-created-by-deprecated
Regarding the other labels, I don't think we use them correctly.
For example I think we should probably have name: opendatahub-operator and part-of: Open Data Hub.
Instance: system does not look unique as well, on the ns we should probably drop it.

Reply:
Yes, we can definitely clean those labels. All the files are auto-generated by the `operator-sdk` when we do 
```
make manifests
```

I can look into how these can be updated